nav2_planner/src/planner_server.cpp

```diff
@@ -167,6 +168,12 @@ PlannerServer::on_activate(const rclcpp_lifecycle::State & state)
     it->second->activate();
   }
 
+  is_path_valid_service_ = shared_from_this()->create_service<nav2_msgs::srv::IsPathValid>(
+    "is_path_valid",
+    std::bind(
+      &PlannerServer::isPathValid, this,
+      std::placeholders::_1, std::placeholders::_2));
+
   auto node = shared_from_this();
```

Comment:
If you move this line above the create_service line, you can use `node` :smile:

Reply:
Done

Reply:
:)